apps/melding-form/src/app/(general)/samenvatting/Summary.test.tsx

```diff
@@ -13,6 +13,10 @@ vi.mock('react', async (importOriginal) => {
   }
 })
 
+vi.mock('./AttachmentsSummary', () => ({
+  AttachmentsSummary: () => <div>Mocked AttachmentsSummary</div>,
```

Comment:
Should we add a test to see if this is rendered at all? This test still passes if we remove `AttachmentsSummary`